micro-rdk-installer/README.md

```diff
@@ -1,12 +1,60 @@
-# The Viam Micro-RDK Installer (IN PROGRESS)
+# The Viam Micro-RDK Installer
 
-When completed, this will be a CLI that allows a user to flash a build of Micro-RDK, along with
-their robot's credentials and their wifi information, directly to their esp32 without requiring
-installation of ESP-IDF, Rust, or Python.
+A CLI that allows a user to flash a build of Micro-RDK, along with their robot's credentials and their wifi information, directly to their esp32 without requiring installation of ESP-IDF, Rust, or Python.
 
-For now, this only has the option of generating a binary of data representing the esp32's Non-Volatile 
-Storage (NVS) Partition and containing wifi and robot credentials. One could then potentially flash just
-the NVS partition to their esp32 using tools provided by ESP-IDF. 
+## Build Instructions
+
+1. `cargo build`
+2. Exwecutable (`micro-rdk-installer`) will be under target/debug
+
+## Usage
+
+```text
+A CLI that can flash a compilation of micro-RDK directly to an ESP32 provided configuration information
+
+Usage: micro-rdk-installer [COMMAND]
+
+Commands:
+  write-flash           Flash a pre-compiled binary with the micro-RDK server directly to an ESP32 connected to
+                            your computer via data cable
+  write-credentials     Write Wi-Fi and robot credentials to the NVS storage portion of a pre-compiled binary
+                            running a micro-RDK server
+  create-nvs-partition  Generate a binary of a complete NVS data partition that conatins Wi-Fi and security
+                            credentials for a robot
+  help                  Print this message or the help of the given subcommand(s)
+
+Options:
+  -h, --help  Print help
+```
+
+### Quick use
+
+1. Find your robot part at https://app.viam.com. Then navigate to the **Setup** tab
+2. Regardless of your operating system, select **Linux** and follow Step 1 to download the Viam app config for your robot
```

Comment:
this is not great (the command as-is drops the file in /etc). 
Mac, Step 2 is what you want and works for all platforms.
Note for reviewers: this will change when we add the relevant tab to app.viam.com setup.